cpp/log/cert.cc

```diff
@@ -38,6 +38,35 @@ static int X509_get_signature_nid(const X509* x) {
 
 namespace cert_trans {
 
+// Useful because util::Status clashes with Cert::Status so we have to
+// refer to it within the namespace while Cert::Status exists.
+static const util::Status& ERROR_STATUS = util::Status::UNKNOWN;
+
+// TODO. These helpers can be removed when Cert::Status has been removed.
+// They make it easier to do an incremental removal
+
+static Cert::Status StatusOrBoolToCertStatus(StatusOr<bool> status) {
```

Comment:
Haha!

If you start from a function that doesn't return a `Cert::Status`, `StatusOrBoolToCertStatus` wouldn't have needed to exist, and if you start from a function that doesn't call anything returning a `Cert::Status`, you wouldn't have needed `CertStatusToStatusOrBool` (but that can be harder, for example in #892, where I'm having to carry around `GetVerifyError` a bit longer, but fewer and fewer code paths are needing it).